i18n-helpers/CHANGELOG.md

```diff
@@ -2,7 +2,15 @@
 
 This lists the most important changes between releases of mdbook-i18n-helpers.
 
-## Version 0.3.4 (2024-05-26)
+## Version 0.3.5 (2024-07-23)
+
+- [#208]: Trim dependencies by @klensy
+- [#209], [#210], [#213], [#214]: Utility for generating diff for PO files by
+  @qwandor
```

Comment:
This isn't part of the `i18n-helpers` crate, it's a separate crate `i18n-report` which has it's own changelog.

Reply:
I missed that, thanks for catching. Updated.